.rubocop_todo.yml

```diff
@@ -33,13 +33,14 @@ Layout/AlignHash:
     - 'app/controllers/storage_locations_controller.rb'
     - 'config/deploy/production.rb'
 
-# Offense count: 1
+# Offense count: 2
```

Comment:
The idea with ratcheting is that numbers in .rubocop_todo.yml can only decrease, never increase

Reply:
Hey @awwaid that was me who did that. It was because we in the middle of the refactor with barcode items changing so I didn't want to have to fix it and then fix it again. Now that Aaron is finished with the PR, I'm happy to take a run at this (and some of the other cops to start getting this down.